lib/tic_tac_toe.ex

```diff
@@ -33,4 +33,12 @@ defmodule TicTacToe do
       _ -> ""
     end
   end
+
+  @spec get_invalid_input_message(atom()) :: String.t()
+  defp get_invalid_input_message(invalid_move_status) do
```

Comment:
Not suggesting a change, I generally stay away from using the word `get` when I'm not dealing with a `Class` or conventions around object oriented language. It helps me shift aways from mental concept of OOP especially when I'm using a functional language like Elixir.

Reply:
That does make a lot of sense actually, I didn't think of the inherent relationship of the word get and OOP, what would you suggest as a better name/is there something better to replace the term `get`? Since I have it in my `board` module as well I think I should change it/remove the `get` in both, but Im not sure of a better substitute. Would something like just removing the get be better?

Reply:
I would suggest just getting rid of `get_`. You can also consider going a step further and have a `Message` module so you can then do `Message.out_of_bound` or `Message.invalid_input`.